.flake8

```diff
@@ -1,16 +1,7 @@
 [flake8]
-ignore =
-  E203,  // Handled by black (Whitespace before ':' -- handled by black)
-  E266,  // Handled by black (Too many leading '#' for block comment)
-  E501,  // Handled by black (Line too long)
-  W503,  // Handled by black (Line break occured before a binary operator)
-  E402,  // Sometimes not possible due to execution order (Module level import is not at top of file)
```

Comment:
ini comments are either `#` or `;` not `//` -- these are actually being interpreted as ignore codes https://flake8.pycqa.org/en/latest/user/configuration.html